app/queries/get_host_content.rb

```diff
@@ -29,17 +29,17 @@ class GetHostContent
     }.freeze
 
     FIELDS = [
-      { field: TABLES[:editions][:id], included_in_group?: true },
-      { field: TABLES[:editions][:title], included_in_group?: true },
-      { field: TABLES[:editions][:base_path], included_in_group?: true },
-      { field: TABLES[:editions][:document_type], included_in_group?: true },
-      { field: TABLES[:editions][:publishing_app], included_in_group?: true },
-      { field: TABLES[:editions][:last_edited_by_editor_id], included_in_group?: true },
-      { field: TABLES[:editions][:last_edited_at], included_in_group?: true },
+      { field: TABLES[:editions][:id], alias: "id", included_in_group?: true },
+      { field: TABLES[:editions][:title], alias: "title", included_in_group?: true },
+      { field: TABLES[:editions][:base_path], alias: "base_path", included_in_group?: true },
+      { field: TABLES[:editions][:document_type], alias: "document_type", included_in_group?: true },
+      { field: TABLES[:editions][:publishing_app], alias: "publishing_app", included_in_group?: true },
+      { field: TABLES[:editions][:last_edited_by_editor_id], alias: "last_edited_by_editor_id", included_in_group?: true },
+      { field: TABLES[:editions][:last_edited_at], alias: "last_edited_at", included_in_group?: true },
       { field: TABLES[:primary_links][:target_content_id], alias: "primary_publishing_organisation_content_id", included_in_group?: true },
       { field: TABLES[:org_editions][:title], alias: "primary_publishing_organisation_title", included_in_group?: true },
       { field: TABLES[:org_editions][:base_path], alias: "primary_publishing_organisation_base_path", included_in_group?: true },
-      { field: TABLES[:statistics_caches][:unique_pageviews], included_in_group?: true },
+      { field: TABLES[:statistics_caches][:unique_pageviews], alias: "unique_pageviews", included_in_group?: true },
       { field: TABLES[:documents][:content_id], alias: "host_content_id", included_in_group?: true },
       { field: TABLES[:editions][:id].count, alias: "instances", included_in_group?: false },
```

Comment:
struggling to follow the whole query now, but is this counting the number of content blocks instead of the `instances` (if that means the amount of appearances in a host)? Shouldn't it be counting the links of type embed on a host (I think we create a new link every time a block is embedded even if it already exists)?

Reply:
ah I see it's covered by a test so must be working :)

Reply:
Yeah, this query returns a row for each link initially, but we get rid of any duplicates with the call to `.group` here https://github.com/alphagov/publishing-api/blob/cd9ae9da86946a5a4933bb27563b0cfbe0eb8077/app/queries/get_host_content.rb#L155